src/victory-util/helpers.js

```diff
@@ -140,6 +140,22 @@ export default {
       {};
   },
 
+  modifyProps(props, fallbackProps) {
+    const themeCheck = props.theme && props.theme.props;
```

Comment:
You probably want to use https://lodash.com/docs#defaults instead of `Object.assign` too